app/src/main/res/values/strings.xml

```diff
@@ -143,6 +155,19 @@
     <string name="account_details_continue" translatable="true">Continue</string>
     <string name="account_details_first_name_error">Please fill in your first name</string>
     <string name="account_details_last_name_error">Please fill in your last name</string>
+    <string name="account_details_created_successfully">Account Created Successfully</string>
+    <string name="account_details_image_upload_error">Error uploading image</string>
+
+    <!-- EmailVerification Strings -->
+    <string name="email_verification_refresh_failed">Failed to refresh</string>
+    <string name="email_verification_sent_failed">Failed to send email</string>
```

Comment:
This is nitpicking as it's a very minor detail, but we should follow a convention of whether or not we should have a point at the end of sentences and keep it throughout the app as we currently have a mix of both

Reply:
You're right we should probably discuss about it. But thankfully the refactoring will help change the sentences easily 👍